.docker/Dockerfile.test

```diff
@@ -10,25 +10,28 @@ FROM govcms8lagoon/test:${GOVCMS_IMAGE_VERSION}
 
 ENV WEBROOT=web
 
+# @todo find out why this.
 RUN mv /app/tests /tests
+
 RUN rm -rf /app
 COPY --from=cli /app /app
 
 # Copy the test scripts.
-COPY .docker/scripts/test/behat /usr/bin/behat
-COPY .docker/scripts/test/phpunit /usr/bin/phpunit
-COPY .docker/scripts/test/drutiny /usr/bin/drutiny
-COPY .docker/scripts/test/lint /usr/bin/lint
-
-RUN chmod +x /usr/bin/behat
-RUN chmod +x /usr/bin/phpunit
-RUN chmod +x /usr/bin/drutiny
-RUN chmod +x /usr/bin/lint
-
-# Note: This is currently disabled until parallel drutiny work is resolved.
-# RUN php -d memory_limit=-1 /usr/local/bin/composer --working-dir=/app/web/sites/all/drutiny/ install --ignore-platform-reqs \
-# && rm -Rf /app/web/sites/all/drutiny/.git \
-# && chmod +x /usr/bin/drutiny
-
-# Since `cli` ran --no-dev, run composer install again to get all dev tools.
-RUN composer install
+RUN cp /app/vendor/govcms/scaffold-tooling/scripts/govcms-deploy /usr/local/bin/
```

Comment:
please change these to run in a single layer

```
RUN cp .... ... \
  && cp ... ... \
  && chmod...
```

Reply:
Maybe a noob question... This image is never stored in the registry or reused by any other images, so wouldn't we prefer a little cleaner instruction set instead of fine-tuning the layer caching strategy?

Reply:
The image is stored locally. This is a standard practice to package things like this in Dockerfiles (see Amazee Dockerfiles, for example).

Reply:
yep i know it's standard practice